mudcontent.ttl

```diff
@@ -0,0 +1,25 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```

Comment:
I added some extra imports. Note that whenever you use the syntax `mud:something` it's actually shorthand for the full URI https://raw.githubusercontent.com/Multi-User-Domain/vocab/main/mud.ttl#something

The prefix statements define what the shorthand is for